packages/aws-cdk/test/diff.test.ts

```diff
@@ -393,15 +393,37 @@ describe('non-nested stacks', () => {
 
     // WHEN
     const exitCode = await toolkit.diff({
-      stackNames: ['A', 'A'],
+      stackNames: ['D'],
```

Comment:
I fixed a broken test. Originally, the test assumed there was no difference, but the specification `stackNames: ['A', 'A']` actually implied a difference existed. The assertion `not.toContain` passed due to the presence or absence of ANSI escape sequences. To correct this, I changed the specification to `stackNames: ['D']`, where no difference exists, and I also removed the ANSI escape sequences from the assertion string.

https://github.com/aws/aws-cdk/blob/10ed1948beb0f83c1b978da9c0a656aa01a382cb/packages/aws-cdk/test/diff.test.ts#L57